bot.py

```diff
@@ -126,13 +121,14 @@ def main(self):
             _end = int(time.time())
             _tot = _end - _start
             _min = min(list_countdown) - _tot
+            print(f"list_countdown: {list_countdown}")
+            print(f"_start: {_start}, _end: {_end}, _tot: {_tot}, _min: {_min}")
             if _min <= 0:
                 continue
 
             self.countdown(_min)
 
-
-if __name__ == "__main__":
+if name == "__main__":
```

Comment:
name? Isn't it __name__?

Reply:
__name__ was throwin syntax error for me

Reply:
i mean `__name__`  not only `name` ?